### PR TITLE
Pipenv with local dev

### PR DESCRIPTION
**Pull request by raymondberg**
## Problem

I want to do a bunch of iterations on the Pipfile but doing that in Docker is very, very tedious.

## Solution

Allow for folks to run Django locally...with caveats that it's unsupported.

### CODE REVIEW

**Comment by raymondberg** [comment]
Relies heavily on commits from #429 , recommend merging that before reviewing.

File: README.md [hunk 1]
```diff
@@ -164,3 +164,13 @@ regardless you can deploy a feature branch with the following command:
...
+
+
+## Alternate Local Development
+
+You can use `make up-services` to start the background services
+locally. Then, if you copy `docker/docker.env` to `.env` you can
+use pipenv to run the service locally: `pipenv run python manage.py runserver`.
+
+If you develop locally you are on your own! All contributions must work in
+docker.
```
Comment (elmq0022) [inline README.md:176]: Maybe add something like: 

Of course, you will need pipenv and the correct python version installed locally.

**Review by elmq0022 — APPROVED** [review]
I really like this one, but I feel a little guilty about that.  I look forward to using standard shell commands and such.  

See my comment about the README.md.  Though if someone doesn't know how to get pipenv and the correct python version they should maybe stick to docker.

File: README.md (hunk 1 above)
Comment (raymondberg) [reply README.md:176]:
```suggestion
docker. This path requires Python and Pipenv installation knowledge.
```

File: docker/Dockerfile [hunk 2]
```diff
@@ -22,9 +23,9 @@ WORKDIR ${SITE_DIR}
 RUN install -d proj/ var/log/ htdocs/ htdocs/static/
 
 # Install python packages
-ADD requirements.txt ${SITE_DIR}requirements.txt
-ADD requirements_test.txt ${SITE_DIR}requirements_test.txt
-RUN pip install -r ${SITE_DIR}requirements_test.txt
+ADD Pipfile ${SITE_DIR}Pipfile
+ADD Pipfile.lock ${SITE_DIR}
```
Comment (raymondberg) [inline docker/Dockerfile:27]:
```suggestion
ADD Pipfile.lock ${SITE_DIR}Pipfile.lock
```

**Pull request merged**